SimulatedWorld_Function.R

```diff
@@ -21,13 +21,30 @@ SimulateWorld <- function(){
     ex <- extent(0.5,20.5,0.5,20.5)
     extent(temp_plain) <- ex
     xy <- coordinates(temp_plain)
-    temp_plain[] <- xy[,2]
+    #temp_plain[] <- xy[,2] # not used
+    # EW: if this is a proxy for bottom temp is 4 deg inc over 100 years too high? 
+    # EW: also these values created a surface with a 4 deg change within a year -- 
+    # -- basically equaivalent to 400-600m depth in the survey
```

Comment:
Yes I don't think this increase is particularly realistic. I think using the downscaled ROMS global climate model outputs will give us realistic change and variability. These fields will be ready ~20 March.